### ISSUE TITLE
v1.1.0 doesn't have a docker image released

### ISSUE THREAD

**Issue by sebinjohn** [mesos/mesos_exporter#85]

**Comment by jpeach** [comment]
@sebinjohn What are you expecting here?

**Comment by sebinjohn** [comment]
@jpeach I assume https://hub.docker.com/r/mesosphere/mesos_exporter/ has the docker images of mesos_exporter.
As per https://hub.docker.com/r/mesosphere/mesos_exporter/tags/ it only has docker images for v1.0.0.
v1.1.0 doesn't have a docker image

This one succeeds
```
docker pull mesosphere/mesos_exporter:v1.0.0
v1.0.0: Pulling from mesosphere/mesos_exporter
90f4dba627d6: Pull complete
5bbc41ca28db: Pull complete
9cf36286db52: Pull complete
dc35fba931bd: Pull complete
Digest: sha256:aef58d3ce332080d542c873a905ff3658404529d94f02bc6f190d4610d3ed8dc
Status: Downloaded newer image for mesosphere/mesos_exporter:v1.0.0
```
and this does not.
```
docker pull mesosphere/mesos_exporter:v1.1.0
Error response from daemon: manifest for mesosphere/mesos_exporter:v1.1.0 not found
```

Is it possible to have docker image released for v1.1.0?

**Comment by jpeach** [comment]
> On Oct 16, 2018, at 3:32 AM, Sebin John <notifications@github.com> wrote:
> 
> @jpeach <https://github.com/jpeach> I assume https://hub.docker.com/r/mesosphere/mesos_exporter/ <https://hub.docker.com/r/mesosphere/mesos_exporter/> has the docker images of mesos_exporter.
> As per https://hub.docker.com/r/mesosphere/mesos_exporter/tags/ <https://hub.docker.com/r/mesosphere/mesos_exporter/tags/> it only has docker images for v1.0.0.
> v1.1.0 doesn't have a docker image
> 
> This one succeeds
> 
> docker pull mesosphere/mesos_exporter:v1.0.0
> v1.0.0: Pulling from mesosphere/mesos_exporter
> 90f4dba627d6: Pull complete
> 5bbc41ca28db: Pull complete
> 9cf36286db52: Pull complete
> dc35fba931bd: Pull complete
> Digest: sha256:aef58d3ce332080d542c873a905ff3658404529d94f02bc6f190d4610d3ed8dc
> Status: Downloaded newer image for mesosphere/mesos_exporter:v1.0.0
> and this does not.
> 
> docker pull mesosphere/mesos_exporter:v1.1.0
> Error response from daemon: manifest for mesosphere/mesos_exporter:v1.1.0 not found
> Is it possible to have docker image released for v1.1.0?
> 
I can look into whether we can push a docker image release into the "mesos" namespace for the next release.

**Comment by elezar** [comment]
I can add a build job in Dockerhub for the tags. Would using the git tag as the docker image tag be sufficient?

**Comment by elezar** [comment]
(there is already a build job for "latest")

**Comment by sebinjohn** [comment]
@elezar The latest tag does not point to the latest release

v1.1.0 has a new command line option `-version`

The following output says the `-version` argument doesn't exist

```
docker run mesosphere/mesos_exporter:latest -version
flag provided but not defined: -version
Usage of mesos-exporter:
  -addr string
        Address to listen on (default ":9105")
  -exportedSlaveAttributes string
        Comma-separated list of slave attributes to include in the corresponding metric
  -exportedTaskLabels string
        Comma-separated list of task labels to include in the corresponding metric
... 

```

**Comment by lloesche** [comment]
I just triggered a manual build. Have to investigate next week why automatic builds don't seem to work currently.

**Comment by sebinjohn** [comment]
The build seems to have updated the `latest` tag
```
docker run mesosphere/mesos_exporter -version
mesos_exporter, version  (branch: , revision: )
  build user:
  build date:
  go version:       go1.11.1
```
It now recognises the `-version` flag.

**Comment by lloesche** [comment]
The breakage happened when we moved `mesosphere/mesos_exporter` -> `mesos/mesos_exporter`. I recreated the image and automatic build job as `mesos/mesos_exporter` (https://hub.docker.com/r/mesos/mesos_exporter/) and triggered a build for the `v1.0.0` and `v1.1.0` tags as well as `latest`.
But that also means the name of the image has changed. I don't see a way of renaming the repo on Dockerhub like you can do on Github.

**Comment by sebinjohn** [comment]
@lloesche Apart from the name change, everything works now.

**Comment by jpeach** [comment]
I just release v1.1.1 to pick up the fix for #86. Looks like Docker Hub automatically picked up the new tag. Thanks for fixing this @lloesche.
